app/src/androidTest/java/com/hedvig/app/feature/onboarding/screens/ChoosePlanScreen.kt

```diff
@@ -0,0 +1,26 @@
+package com.hedvig.app.feature.onboarding.screens
+
+import android.view.View
+import com.agoda.kakao.check.KCheckBox
+import com.agoda.kakao.recycler.KRecyclerItem
+import com.agoda.kakao.recycler.KRecyclerView
+import com.agoda.kakao.screen.Screen
+import com.agoda.kakao.text.KButton
+import com.agoda.kakao.text.KTextView
+import com.hedvig.app.R
+import org.hamcrest.Matcher
+
+class ChoosePlanScreen : Screen<ChoosePlanScreen>() {
```

Comment:
Since we migrated to Kaspresso, we can write this in this way instead:

```suggestion
object ChoosePlanScreen : KScreen<ChoosePlanScreen>() {
	override val layoutId = R.layout.activity_choose_plan
	override val viewClass = ChoosePlanActivity::class.java
	// ...
```

Then we can use it as such:

```kotlin
	ChoosePlanScreen {
		recycler {
			// ...
		}
	}
```